sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ProgressReporter.scala

```diff
@@ -201,7 +201,7 @@ trait ProgressReporter extends Logging {
 
     if (hasExecuted) {
       // Reset noDataEventTimestamp if we processed any data
-      lastNoExecutionProgressEventTime = Long.MinValue
+      lastNoExecutionProgressEventTime = triggerClock.getTimeMillis()
```

Comment:
When reset `"lastNoExecutionProgressEventTime =Long.MinValue"`, it will make `"now - noDataProgressEventInterval >= lastNoExecutionProgressEventTime"` always `true` when there is no new data. Then progress reporter will report an `empty` progress.